bitnami/redis/templates/_helpers.tpl

```diff
@@ -231,6 +231,15 @@ Return Redis&reg; password
 {{- end }}
 {{- end }}
 
+{{- define "common.secrets.get" -}}
```

Comment:
Please add a comment explaining what this function does. There are several examples in this file already.